_sips/minutes/2020-03-11-minutes.md

```diff
@@ -0,0 +1,220 @@
+* Drop procedure syntax
+    * Guillaume: it's dead
+* Drop Symbol literals
+    * Seb: it's basically done
+* Drop XML literals
+    * Guillaume: 3.0 supports XML literals (under `-Scala2` flag?)
+    * Seb: it doesn't leak into binaries/TASTy, can remove later
+    * Nic: we have a replacement but just without unapply/pattern matching
+    * Martin: and that isn't used much, so you'd just need to do something else there
+* Eta expansion
+    * no `_` any more
+    * Gui: issue with println, you `println(myMethod)` and it's not a type error (b/c no args) it prints the function
+    * Martin: multi-verse equality fixes `==` for functions, previously always `false`
+* Implicit conversions with Conversion
+    * Gui: from threads, used a lot by DSL libraries
+    * <https://contributors.scala-lang.org/t/proposal-changes-to-implicit-conversions/4101/7>
+* Implicit resolution changes
+    * Seb: so many changes
+    * Gui: need to discuss them 1 by 1, important to discuss them individually
+    * Gui: some are common sense, some are for ambiguity, adding parameters (prioritisation)
+    * Martin: would be good to discuss the details with Miles
+    * Seb: shouldn't be in the "easy" list, then
+* Intersection/union
+    * Seb: no choice, and it's good
+* Opaque type aliases
+    * Seth: pushback from contributors, don't ship, wait for Valhalla
+    * Seb: they're not to replace value classes.  They have the runtime characteristics of type aliases.
+    * Lukas: can it be ported back to nsc?  Gui: no, sadly
+* Open classes
+    * Seb: summary: overall seems good, but details
+    * Seb: should it be scoped at the package? when should we warn/error?
+    * Gui: seems not easy
+    * Martin is against the counter-proposal of having it as an opt-in annotation
+* Parameter untupling
+    * Gui: the strong version is that anything on the left is a pattern (rather than just tuple)
+    * Seth: not crazy about it, seems like another thing you need to learn
+    * Gui: on the contrary, the status quo (having to add a `case`) forces you to learn this detail early
+    * Seb: but we're dropping auto-tupling?
+    * Martin: it's related to the pattern bindings changes (below)
+* Pattern bindings
+    * Martin `for (case Foo(a, b) <- ...)` for filtering, and val pattern, same
+    * https://dotty.epfl.ch/docs/reference/changed-features/pattern-bindings.html
+* Toplevel definitions
+    * Gui: details around `private`. top-level `private type` and `private class`, the type is in a synthetic object, the class is really top level
+    * that also impacts opaque types
+    * Seb: then let's force users to put in object and export instead
+    * but, boilerplate...
+* Trait parameters
+    * Gui/Seb: yes, everyone wants them
+    * Gui: something about mixing in multiple traits and them having implicit parameters (that need hand-wiring?)
+* Type lambdas
+    * Gui/Seb: yes, everyone wants them
+    * Gui: something about variance
```

Comment:
add:
* Seth: I'll look into whether kind-projector is on track with the changes on their end